pkg/dependency/parser/java/pom/testdata/with-spaces/pom.xml

```
@@ -0,0 +1,27 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion> 4.0.0</modelVersion>
+
+    <groupId>com.example </groupId>
+    <artifactId> root-pom-with-spaces </artifactId>
+    <version> 1.0.0</version>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId> org.example</groupId>
+                <artifactId>example-dependency </artifactId>
+                <version>1.2.4 </version>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
```

This PR doesn't allow its maintainers to change it, so I will ask you to add the property to the test file:
```suggestion
    <properties>
        <dependency.version > 1.2.4</dependency.version >
    </properties>

    <dependencyManagement>
        <dependencies>
            <dependency>
                <groupId> org.example</groupId>
                <artifactId>example-dependency </artifactId>
                <version>${dependency.version} </version>
            </dependency>
        </dependencies>
    </dependencyManagement>

```

PS you will need to change line numbers `org.example:example-nested:3.3.3`

---

@DmitriyLewen I updated the file as requested.

As for the maintainer edit, I am surprised as it is something I usually let on by default but it seems I don't even have the option to change it on this PR even following [GitHub's documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).

---

You need to use `${dependency.version} ` property in `example-dependency`:
https://github.com/aquasecurity/trivy/blob/e79f0e794f76a0023c42b89fa0583ab70d642140/pkg/dependency/parser/java/pom/testdata/with-spaces/pom.xml#L14

> The fork is under an organization, that's why.

hm... I didn't know that, thanks for the new information for me!

---

Sorry ! I misread your code suggestion. Fixed